<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableWiget</class>
 <widget class="QMainWindow" name="tableWiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tableWiget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item row="0" column="1">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="0" column="2">
       <widget class="QColumnView" name="columnView"/>
      </item>
      <item row="1" column="1">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="1" column="2">
       <widget class="QUndoView" name="undoView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
